data/src/main/scala/ch.epfl.scala.index.data/elastic/SeedElasticSearch.scala

```diff
@@ -22,30 +22,34 @@ class SeedElasticSearch(paths: DataPaths, githubDownload: GithubDownload)(
   def run(): Unit = {
 
     val exists = Await
-      .result(esClient.execute { indexExists(indexName) }, Duration.Inf)
-      .isExists()
+      .result(
+        esClient.execute(indexExists(indexName)),
+        Duration.Inf
+      )
+      .isExists
 
     if (exists) {
-      Await.result(esClient.execute {
-        deleteIndex(indexName)
-      }, Duration.Inf)
+      Await.result(
+        esClient.execute(deleteIndex(indexName)),
+        Duration.Inf
+      )
     }
 
     val projectFields = List(
-      keywordField("organization")
-        .normalizer("lowercase")
+      textField("organization")
```

Comment:
That seems counterintuitive to me. How come is it better as a text field rather than a keyword?

Reply:
I agree it is counter-intuitive for this one but there is an explanation
As soon as we want to search with text that is inputted by a user, it is better to use a `textField`. If I type `typelevel cats` I won't match the `typelevel` keyword but I would match the `typelevel` text field.
`keywordField` should be used for exact term matching and aggregations only.

Reply:
Another example: if I type `scala js` I would like to match the `scala-js` organization and the `scala-js` repo.